src/components/navbar/Navbar.jsx

```diff
@@ -0,0 +1,35 @@
+import "../navbar/navbar.css";
+import { useState } from "react";
+import classNames from "classnames";
+
+const Navbar = () => {
+    const [isOpen, setIsOpen] = useState(false);
+    const openNav = () => {
+        setIsOpen(!isOpen)
+    }
+return(
+    <div>
+        <nav className="navbar-ecom">
+            <a href="../index.html" className="brand-title-ecom">fastcart</a>
```

Comment:
use link tag - react-router-dom for navigation

Reply:
I will add routers once i finished up with the UI part.